sdk/monitor/ci.yml

```diff
@@ -30,5 +30,5 @@ extends:
     Artifacts:
     - name: azure_mgmt_monitor
       safeName: azuremgmtmonitor
-    - name: opentelemetry-exporter-azuremonitor
+    - name: opentelemetry_exporter_azuremonitor
```

Comment:
What's this?

Reply:
this is to set up the public ci - this should go away once we rebase with master.